src/rollup.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./components";
+export { default as markurzExtensionTheme } from "./theme";
```

Comment:
I guess this is normal, but absolute import doesn't work isn't it ?

Reply:
Indeed, `index.ts` files all refer to each other according to relative paths, to allow the exports to be properly built.